cloud/shared/bin/lib/terraform.py

```diff
@@ -144,7 +144,7 @@ def perform_apply(
     # Enable compact-warnings as we have a bunch of
     # "value of undeclared variables" warnings as some variables used in one
     # deployment (e.g. aws) but not the other.
-    terraform_apply_cmd = f'terraform -chdir={terraform_template_dir} apply -input=false -var-file={tf_vars_filename} -compact-warnings'
+    terraform_apply_cmd = f'terraform -chdir={terraform_template_dir} apply -input=false -var-file={tf_vars_filename} -compact-warnings -auto-approve'
```

Comment:
**question:** Would this cause the changes to be applied without confirming? We can add that as a variable, but we should make the default that they have to approve any changes.

Reply:
This was leftover from my debugging efforts. Removed it because I have a SKIP_CONFIRMATION variable in my azure config file that adds auto-apply. I've found that setup and deployment fails without auto-apply. Still debugging why that is. I've added a recommendatoin to add this variable to my azure deployment notes.